### DISCUSSION TITLE
how to add tables to the page - which markdown is correct?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#51786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to add tables to the page - which markdown is correct?

```

| ------------ | ------------ | ------------ |
|   |   |   |
|   |   |   |
|   |   |   |

```

```
Spalte 1 | Spalte 2 | Spalte 3
-------- | -------- | --------
Inhalt   | Inhalt   | Inhalt
Inhalt   | Inhalt   | Inhalt

```
see the page: where i want to get this done 

https://github.com/fsj-digital
https://fsj-digital.github.io/DigitalHub/

look forward to hear from you 

regards

**Comment by radar-blip** [comment]
Your second example will render like this using GitHub Flavored Markdown.

![Screenshot from 2023-04-02 22-01-40](https://user-images.githubusercontent.com/39196767/229395556-47dfccc1-9f04-4d7f-9e22-8a64943b62c6.png)
